TestsProject/TestsProjectTests/InitializationTests.m

```diff
@@ -53,6 +54,26 @@ - (BOOL)resetEngine;
         it(@"Should check version", ^{
             [[[G8Tesseract version] should] equal:@"3.03"];
         });
+        
+        it(@"Should clear cache", ^{
+            // while recognition is in progress
+            for (int i = 0; i <= 10; i++) {
+                G8RecognitionOperation *operation = [[G8RecognitionOperation alloc] init];
+                operation.tesseract.image = [UIImage imageNamed:@"well_scaned_page"];
+                operation.tesseract.language = kG8Languages;
+
+                NSOperationQueue *queue = [[NSOperationQueue alloc] init];
+                [queue addOperation:operation];
+            }
+            
+            [[theBlock(^{
+                [G8Tesseract clearCache];
```

Comment:
Could you split these test for "Should not raise on cache clearing" and "Should clear cache on memory warning"?